libraries/platforms/dromajo-vcs/software/src/crt0.S

```diff
@@ -0,0 +1,91 @@
+# Start code
+# This assembly file mimics the BlackParrot perch library start.S
+# with minor modifications to allow arguments to be loaded into the
+# corresponding registers for main
```

Comment:
Should this be in the BP SDK? 

Seems odd to have here, because we don't really depend on the toolchainr.

Reply:
Yeah I think this can go in the BP SDK once we support argparsing there (that's the only real difference between the two). In the interest of merging I would say put a note for deprecation

Reply:
Gotcha Dan.